package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "5.7.1",
+  "version": "5.8.0",
```

Comment:
@severinbeauvais OK with you to consider this a minor version release?

Reply:
We just went up a minor version already but, by my own logic, yes, this is fine 😄